views/index.erb

```diff
@@ -116,9 +116,7 @@
                       ['1', 'Authentication only (default)'],
                       ['2', 'Identity-verified'],
                       ['0', 'IALMax'],
-                      if ENV['vtr_disabled'] != 'true'
-                        ['biometric-comparison-required', 'Biometric Comparison']
-                      end,
+                      ['biometric-comparison-required', 'Biometric Comparison']
```

Comment:
to match https://github.com/18F/identity-oidc-sinatra/pull/158 it needs some new copy right?
```suggestion
                      ['biometric-comparison-required', 'Biometric Comparison (disabled in prod)']
```

Reply:
Yes, I failed to hit "Save" it seems